index.html

```diff
@@ -26,6 +26,19 @@
       <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
+	
+	<script>
+		function get_action(form) {
+
+		var v = grecaptcha.getResponse();
+		if(v.length == 0)
+		{
+			document.getElementById('captcha').innerHTML="You left the Captcha Code empty";
```

Comment:
Using an ID here might not work, because there's more than one on the page. Needs testing though, once we've got the key added.